pkg/types/jar/v0.0.1/entry_test.go

```diff
@@ -0,0 +1,229 @@
+/*
+Copyright © 2021 The Rekor Authors
```

Comment:
should these all say sigstore?

Reply:
I had this question - should we set this org-wide in all projects to just be "sigstore authors"?

Reply:
I think @cpanato is planning to do this, he already made a couple of PRs to unity the license header. 

I think the name thing originates from when I first started the project with just me in a personal repo.

Reply:
Yes, doing for the missing repos today or tomorrow (15.04.2021) :)